packages/core/src/config/load.ts

```diff
@@ -41,7 +41,7 @@ export async function loadConfig(configPath?: string, customExtends?: string[]):
       },
       //support redocly.com domain for future compatibility
```

Comment:
It seems that the comment need to be updated.
~Do I understand it correctly, that we don't need similar config for 'eu' region?~
(ok, I see, EU domain is already `.com`)